### PR TITLE
Book repository

### PR DESCRIPTION
**Pull request by juandbc**
## Summary
* Implemented the book repository

## Trello Card
https://trello.com/c/rC90Hbj9/15-crear-un-repositorio-de-books

### CODE REVIEW

File: src/main/java/wolox/training/repositories/BookRepository.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package wolox.training.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import wolox.training.models.Book;
+
+@Repository
+public interface BookRepository extends JpaRepository<Book, String> {
```
Comment (Gianni-Nicolas) [inline src/main/java/wolox/training/repositories/BookRepository.java]: When you make the changes in Book, remember to change the id of the repository (String) for the new data type (long).
```suggestion
public interface BookRepository extends JpaRepository<Book, Long> {
```

**Review by Gianni-Nicolas — CHANGES_REQUESTED** [review]

**Review by jeymarsonwolox — COMMENTED** [review]
change the origin of this branch to the book-model branch

File: src/main/java/wolox/training/repositories/BookRepository.java (hunk 1 above)
Comment (juandbc) [reply src/main/java/wolox/training/repositories/BookRepository.java]: In this case, the id should be String type because the ISBN attribute it is the id

Comment (juandbc) [reply src/main/java/wolox/training/repositories/BookRepository.java]: I didn't check the #3 when I see this. Now I understand the change

**Review by Gianni-Nicolas — APPROVED** [review]

**Review by jeymarsonwolox — APPROVED** [review]

**Pull request merged**